packages/app/src/pages/index.tsx

```diff
@@ -19,10 +22,12 @@ const HomePage: NextPage = () => {
       <div className="flex-grow flex flex-col gap-4 items-center justify-center p-8 pb-[50vh]">
         <h1 className="text-4xl">Example NFT</h1>
 
-        <p>
-          {totalSupply.data?.toNumber().toLocaleString() ?? "??"}/
-          {maxSupply.data?.toNumber().toLocaleString() ?? "??"} minted
-        </p>
+        {isMounted && (
```

Comment:
Small nit: I tend to prefer full ternaries like `{condition ? (<>render</>) : null}` as it avoids the edge case where your condition might not be a boolean and will unintentionally render the condition value itself.

Not sure if there is a linter for this kind of thing. I feel like I wrote one for this pattern at Stripe but I don't have the code anymore.

Reply:
@holic Cool, I hadn't thought about this much before but I agree. I wasn't able to find an existing rule for this, although there is one proposed here that we might be able to use: https://github.com/jsx-eslint/eslint-plugin-react/pull/2888

I made https://github.com/holic/web3-scaffold/issues/29 so we can track this in backlog